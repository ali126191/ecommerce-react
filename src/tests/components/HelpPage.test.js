import React from 'react';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';
import HelpPage from '../../components/HelpPage';

test('should render HelpPage correctly', () => {
    const wrapper = shallow(<HelpPage/>)
    expect(toJSON(wrapper)).toMatchSnapshot()
})