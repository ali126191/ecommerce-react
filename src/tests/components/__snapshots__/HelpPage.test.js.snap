// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render HelpPage correctly 1`] = `
<div>
  <Header />
  <h2>
    Help
  </h2>
</div>
`;
